backend/src/packages/journal-entries/journal-entry.service.ts

```diff
@@ -29,21 +49,57 @@ class JournalEntryService implements Service {
     };
   }
 
-  public async create(
-    payload: JournalEntryCreateRequestDto,
-  ): Promise<JournalEntryGetAllItemResponseDto> {
+  public async create({
+    body,
+    userId,
+  }: CreateJournalEntryPayload): Promise<JournalEntryGetAllItemResponseDto> {
+    const user = await userService.findById(userId);
+
+    if (!user) {
+      throw new UsersError({
+        status: HTTPCode.NOT_FOUND,
+        message: ExceptionMessage.USER_NOT_FOUND,
+      });
+    }
```

Comment:
As we are using auth user, the prehandler decoded token to userId and found user, is it a mandatory thing to make this one here

Reply:
Ok, I will remove it.